.devcontainer/postCreate.sh

```diff
@@ -9,6 +9,12 @@ pip3 install --user -r .devcontainer/requirements.in
 #set RStudio working directory
 ! grep -q `pwd` ~/.config/rstudio/rstudio-prefs.json && cat ~/.config/rstudio/rstudio-prefs.json | jq ". + {\"initial_working_directory\":\"`pwd`\"}" >  ~/.config/rstudio/rstudio-prefs.json 
 
+#disable rstudio server auth
+! grep -q "auth-none=1" /etc/rstudio/rserver.conf && sudo tee -a /etc/rstudio/rserver.conf <<< "auth-none=1"
+
+#set default rstudio user
+! grep -q "USER=rstudio" /etc/environment && sudo tee -a /etc/environment <<< "USER=rstudio"
```

Comment:
Is this something we could move to the Dockerfile instead?

Reply:
Good point, yes we could and should. I'll do that now

Reply:
Done